.rubocop_todo.yml

```diff
@@ -451,13 +451,6 @@ Security/YAMLLoad:
 Style/Alias:
   Enabled: false
 
-# Offense count: 10
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
-# SupportedStyles: always, conditionals
-Style/AndOr:
-  Enabled: false
```

Comment:
I think we should enforce this one only on conditionals. `do_something and exit(1)` and such are common idioms

Reply:
I agree with @tbrisker, `&&` and `and` are different things, one is for condition second is for flow control, the priority of evaluation  is different